### ISSUE TITLE
Rule proposal: merge conflict hint code

### ISSUE THREAD

**Issue by zzzzzzboy** [sindresorhus/eslint-plugin-unicorn#1454]
## When resolving merge conflicts, You might forget to delete the merge conflict prompt code string

![image](https://user-images.githubusercontent.com/42526781/127255426-2cc58e42-b1c5-406c-89a0-c7a21602950a.png)
```js
>>>>>>> 2sdwe5sad1551we451sdwe5412ase54asd21we5a
```

## Pass

Forbid it. If a string starting with ">>>>>>>" is detected, an error message is displayed

**Comment by fisker** [comment]
It's actually a good idea, but problem is the parser may throw errors when parsing, are you closing for this reason?

**Comment by zzzzzzboy** [comment]
> It's actually a good idea, but problem is the parser may throw errors when parsing, are you closing for this reason?

In fact, there were no errors between the time I submitted the code and the time I released it.
I closed it because I wasn't sure if I should raise the issue here or under ESLint's official gitHub.
